hw/ip/kmac/lint/sha3.waiver

```diff
@@ -18,3 +18,8 @@ waive -rules {HIER_NET_NOT_READ NOT_READ} -location {keccak_round.sv} \
 waive -rules {INPUT_NOT_READ} -location {keccak_2share.sv} \
       -regexp {'sel_i' is not read from} \
       -comment "Unmasked version does not use sel_mux signal"
+
+waive -rules {TWO_STATE_TYPE} -location {keccak_2share.sv} \
+      -regexp {'index_z' is of two state} \
+      -comment "index_z behaves as constant"
+      
```

Comment:
It seems you have a space or something on the last line.

Reply:
Revised